src/pages/patientView/PatientViewPageUtils.tsx

```diff
@@ -4,6 +4,42 @@ import {
     IGenePanelDataByProfileIdAndSample,
     isSampleProfiledInProfile,
 } from 'shared/lib/isSampleProfiled';
+import { GenericAssayMeta } from 'cbioportal-ts-api-client';
+import {
+    IMutationalSignature,
+    IMutationalSignatureMeta,
+    IMutationalCounts,
+} from 'shared/model/MutationalSignature';
+import {
+    getGenericAssayMetaPropertyOrDefault,
+    getGenericAssayCategoryFromName,
+} from 'shared/lib/GenericAssayUtils/GenericAssayCommonUtils';
+
+type MutationalSignatureLabelMap = {
```

Comment:
these types should be moved into shared/model folder

Reply:
Moved them to shared/model/MutationalSignature.ts